content/scss/styleguide-customizations.scss

```diff
@@ -122,6 +122,13 @@
   }
 }
 
+/* Footer margins
+   ========================================================================== */
+
+.br-componentgroup-footer-wrapper {
```

Comment:
@Wolfr I notice this is a class used in the `core`. Maybe we have to document somewhere in Bedrock that this is the right way (once we are able to remove the core from the repository).